src/modules/75-cd/components/PipelineSteps/DeployEnvironmentEntityStep/DeployInfrastructure/DeployInfrastructure.tsx

```diff
@@ -226,9 +226,13 @@ export default function DeployInfrastructure({
           })
         }
       } else if (isMultiInfrastructure && isEmpty(selectedInfrastructures)) {
-        // set value of unique path to All in case no infrastructures are selected
+        // set value of unique path to All in case no infrastructures are selected or runtime if set to runtime
```

Comment:
typo - infrastructures
```suggestion
        // set value of unique path to All in case no infrastructures are selected or runtime if infrastructures is set to runtime
```